workflows/chipseq.nf

```diff
@@ -484,7 +486,7 @@ workflow CHIPSEQ {
                         meta.replicates_exist = groups.max { groups.value }.value > 1
                         [ meta, peaks ] }
                 .set { ch_antibody_peaks }
-
+            // ch_antibody_peaks.dump()
```

Comment:
```suggestion
```

Reply:
Fixed in [277bd11](https://github.com/nf-core/chipseq/pull/279/commits/277bd119f4747960d0059df30f7a3cea8fccb625)